apps/wallet/src/ui/app/redux/slices/sui-objects/index.ts

```diff
@@ -169,8 +169,22 @@ const slice = createSlice({
             .addCase(
                 fetchAllOwnedAndRequiredObjects.rejected,
                 (state, { error: { code, name, message } }) => {
+                    const matchesHtml =
```

Comment:
Let's not parse the HTML. We can just use the HTTP error code to drive the error message. Ideally we can get error responses to be JSON (can follow-up with PE on this if we care a lot).

Reply:
The only way to get the HTTP status is from the message, doing something like this with the regex. Sdk doesn't expose it.

Reply:
I'd rather we make the change to the SDK to expose the error code than do this.

Reply:
In the end I just removed the html response from the error in the sdk. Probably we need to somehow improve the errors from the sdk to include either the request error and/or better/more info. But that's a much bigger task and I don't think now it's the best time to do it.